docs/integrations/gitea/discovery.md

```diff
@@ -0,0 +1,63 @@
+---
+id: discovery
+title: Gitea Discovery
+sidebar_label: Discovery
+# prettier-ignore
+description: Automatically discovering catalog entities from Gitea repositories
+---
+
+:::info
+This documentation is written for [the new backend system](../../backend-system/index.md) which is the default since Backstage [version 1.24](../../releases/v1.24.0.md).
```

Comment:
Let's remove this, it's been the default for a long time now. We plan to remove this from the other pages in the next few months as well.